libqtile/widget/keyboardlayout.py

```diff
@@ -43,6 +44,7 @@ class KeyboardLayout(base.InLoopPollText):
         ("configured_keyboards", ["us"], "A list of predefined keyboard layouts "
             "represented as strings. For example: "
             "['us', 'us colemak', 'es', 'fr']."),
+        ("options", None, "list of setxkbmap options. Ex., 'compose:menu,grp_led:scroll'"),
```

Comment:
`list` may not be the best description of this, as it is really a string. It may be worth changing this to `additional options passed to setxkbmap via -options`

Reply:
The option name is `-option`. It's correctly written `-option` in the code though :slightly_smiling_face:

Reply:
You are right. At first I thought to add this as list in real, but checked how it is with `setxkbmap` displayed and `options` become string. Fixed. This diff was in my mind for two years, just time is good to do all small stuffs because I had to build config from the begining